apis/core/lib/domain/identifiers.ts

```diff
@@ -21,3 +21,7 @@ export function csvSourceCollection(csvMapping:GraphPointer<NamedNode>): NamedNo
 export function csvSource(project: GraphPointer<NamedNode>, fileName: string): NamedNode {
   return $rdf.namedNode(`${project.value}/csv-source/${url.slugify(fileName)}`)
 }
+
+export function associatedMedia(csvSource: GraphPointer<NamedNode>): NamedNode {
+  return $rdf.namedNode(`${csvSource.value}/media/`)
+}
```

Comment:
Is this used?

Besides, I think that we can just keep a blank node for the `MediaObject`